scripts/scil_bundle_label_map.py

```diff
@@ -2,47 +2,93 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute the label image (Nifti) from a centroid and tractograms (all
-representing the same bundle). The label image represents the coverage of
-the bundle, segmented into regions labelled from 0 to --nb_pts, starting from
-the head, ending in the tail.
+Compute label image (Nifti) from bundle(s) and centroid(s).
+Each voxel will have a label that represents its position along the bundle.
 
-Each voxel will have the label of its nearest centroid point.
+The number of labels will be the same as the centroid's number of points,
+unless specified otherwise.
 
-The number of labels will be the same as the centroid's number of points.
+# Single bundle case
+  This script takes as input a bundle file, a centroid streamline corresponding
+  to the bundle. It computes label images, where each voxel is assigned the
+  label of its nearest centroid point. The resulting images represent the
+  labels, distances between the bundle and centroid.
+
+# Multiple bundle case
+  When providing multiple (co-registered) bundles, the script will compute a
+  patch-wise correlation map between density maps as a proxy for confidence in
+  the bundle's reconstruction.
+
+  The correlation map can be thresholded to remove low confidence regions.
+  It will also compute the labels maps for after concatenating bundles,
+  ensuring that the labels are spatially consistent between bundles.
+
+# Hyperplane method
+  The default is to use the euclidian/centerline method, which is fast and
+  works well for most cases.
+
+  The hyperplane method allows for more complex shapes and to split the bundles
+  into subsections that follow the geometry of each kind of bundle.
+  However, this method is slower and requires extra quality control to ensure
+  that the labels are correct. This method requires a centroid file that
+  contains multiple streamlines.
+
+  This method is based on the following paper [1], but was heavily modified
+  and adapted to work more robustly across datasets.
+
+# Manhattan distance
+  The default distance (to barycenter of label) is the euclidian distance.
+  The manhattan distance can be used instead to compute the distance to the
+  barycenter without stepping out of the mask.
+
+Colormap selection affects tractograms coloring for visualization only.
+For detailed information on usage and parameters, please refer to the script's
+documentation.
 
 Formerly: scil_compute_bundle_voxel_label_map.py
+
+Author:
+-------
+Francois Rheault
+francois.m.rheault@usherbrooke.ca
+
+------------------------------------------------------------------------------------------
+Reference:
```

Comment:
We usually put the references as EPILOG in argparse.

Reply:
@arnaudbore What is the way now?
Because now it seems that EPILOG is used for version_string
(from scilpy.version import version_string)